### PR TITLE
[cli] Update the version to 1.2.1

### PR DESCRIPTION
**Pull request by daniele77**
Update cli library version to 1.2.1

### CODE REVIEW

File: ports/cli/portfile.cmake [hunk 1]
```diff
@@ -3,8 +3,8 @@ include(vcpkg_common_functions)
 vcpkg_from_github(
```
Comment (NancyLi1013) [inline ports/cli/portfile.cmake:3]: `include(vcpkg_common_functions)` is no longer needed here. So please remove this line.

File: ports/cli/portfile.cmake [hunk 2]
```diff
@@ -3,8 +3,8 @@ include(vcpkg_common_functions)
...
     OUT_SOURCE_PATH SOURCE_PATH
     REPO daniele77/cli
-    REF v1.2.0
-    SHA512 ff548cbc1a77ded32f67d0ff4740d2abb31226cb6f0d9d431e1a35dcdfcaf68a2b9e16e926fc88f19aa17f5c6f5f8e2aead83ff65d7557c192bdd7d4ce2a2d3e 
+    REF v1.2.1
+    SHA512 fc834608ac2ca6c3f330711c80897cc581eb1ed8da7a0d56a07efe6650d871f5317c777a6a029fdc0177684f08479bd129cc0ce748354bac3dbe6af79a390d14 
     HEAD_REF master
 )
```
Comment (NancyLi1013) [inline ports/cli/portfile.cmake:10]: Could you please update

`file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/cli RENAME copyright)`

as

`file(INSTALL ${SOURCE_PATH}/LICENSE DESTINATION ${CURRENT_PACKAGES_DIR}/share/${PORT} RENAME copyright)`?

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

File: ports/cli/portfile.cmake (hunk 2 above)
Comment (daniele77) [reply ports/cli/portfile.cmake:8]: done

File: ports/cli/portfile.cmake (hunk 1 above)
Comment (daniele77) [reply ports/cli/portfile.cmake:1]: done

**Review by NancyLi1013 — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for your contribution!

**Pull request merged**